CHANGELOG.md

```diff
@@ -6,6 +6,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.22.0] - March 3, 2021
+
+### Design kit
+#### Added
+- Date field and date picker
+#### Fixed
+- Incorrect text color in TextInput Disabled state
+
+### React Components
+#### Added
+- New Date Input component with Datepicker
```

Comment:
I guess we could write this "date input with date picker" as in documentation. My bad, 👋 I used different format 😁

Reply:
Otherwise it is good to go